modules/vaos/app/models/vaos/appointment_request_form.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+
+require 'active_model'
+require 'common/models/form'
+
+module VAOS
+  class AppointmentRequestForm < Common::Form
+    attribute :email, String
+    attribute :phone_number, String
+    attribute :option_date1, String # TODO: ideally these would be better abstracted as an array for a FE consumer
```

Comment:
Please either address this TODO before merge or extract to an issue and remove comment lest we have further tech debt :D

Reply:
@jbalboni what are your thoughts on these attributes? We could have you provide it to us in a  better way and then break it out like VAMF needs as part of re-serialization. For now I kept it like this because any errors VAMF returns would otherwise not be particularly sensible. That being said, if we're doing our own validations before making the request it shouldn't be a problem.

Reply:
I would leave it as is. I think we should eventually completely restructure this data, rather than doing one off changes here and there.

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3685